src/v3/src/util/theme.ts

```diff
@@ -162,82 +160,10 @@ export const createTheme = (
-    set(theme, 'typography.fontFamily', mergedTokens.TypographyFamilyBody);
-    set(theme, 'typography.fontWeightBold', mergedTokens.TypographyWeightBodyBold);
-    set(theme, 'typography.fontWeightRegular', mergedTokens.TypographyWeightBody);
-    set(theme, 'typography.h1.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h1.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h1.fontSize', mergedTokens.TypographySizeHeading1);
-    set(theme, 'typography.h1.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h1.lineHeight', mergedTokens.TypographyLineHeightHeading1);
-    set(theme, 'typography.h2.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h2.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h2.fontSize', mergedTokens.TypographySizeHeading2);
-    set(theme, 'typography.h2.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h2.lineHeight', mergedTokens.TypographyLineHeightHeading2);
-    set(theme, 'typography.h3.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h3.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h3.fontSize', mergedTokens.TypographySizeHeading3);
-    set(theme, 'typography.h3.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h3.lineHeight', mergedTokens.TypographyLineHeightHeading3);
-    set(theme, 'typography.h4.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h4.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h4.fontSize', mergedTokens.TypographySizeHeading4);
-    set(theme, 'typography.h4.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h4.lineHeight', mergedTokens.TypographyLineHeightHeading4);
-    set(theme, 'typography.h5.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h5.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h5.fontSize', mergedTokens.TypographySizeHeading5);
-    set(theme, 'typography.h5.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h5.lineHeight', mergedTokens.TypographyLineHeightHeading5);
-    set(theme, 'typography.h6.color', mergedTokens.TypographyColorHeading);
-    set(theme, 'typography.h6.fontFamily', mergedTokens.TypographyFamilyHeading);
-    set(theme, 'typography.h6.fontSize', mergedTokens.TypographySizeHeading6);
-    set(theme, 'typography.h6.fontWeight', mergedTokens.TypographyWeightHeading);
-    set(theme, 'typography.h6.lineHeight', mergedTokens.TypographyLineHeightHeading6);
-    set(theme, 'typography.overline.lineHeight', mergedTokens.TypographyLineHeightOverline);
-    set(theme, 'typography.subtitle1.fontFamily', mergedTokens.TypographyFamilyBody);
-    set(theme, 'typography.subtitle2.fontFamily', mergedTokens.TypographyFamilyBody);
```

Comment:
Are these no longer needed? Are they added by Odyssey instead?

Reply:
Yes these aren't needed anymore. They were being set because we wanted to wait until Odyssey shipped an official default theme and token override support with 1.x. 

Now, all of these `theme` values derive their default from the 1.x theme and we inject any token overrides of this `theme` when we construct it, see: https://github.com/okta/okta-signin-widget/pull/3471#discussion_r1401286170